packages/documentation/src/pages/documentation-guide/documentation-style-guide.mdx

```diff
@@ -0,0 +1,134 @@
+- Use clean markdown
+  - Indent with 2 spaces
+  - Add a space between different markdown types
+
+## Emphasis
+
+- **Double astericks** for `**bold**`
+- **Underscore** for `_italics_`
+- **Greater than** for `> block quotes`
+- Make documents easy to scan by emphasizing **subjects**, **actors** and **verbs**
+
+## Lists
+
+- Use **numbered lists** for **sequences**
+  - Number lists with 1. for every step
+- Use **bulleted lists** for most **other lists**
+- **Alphabetize** when not using another type of order
+- **Indent** subitems with **2 spaces**
+
+## Code
+
+- **Prefer code blocks** to inline code
+- **Fence code blocks** with **three back ticks** and language for syntax highlighting
+  - **Recommended**:
+
+>- Install yarn:
+```bash
+yarn install
+```
+
+  - **Not recommended**:
+>Run `yarn install`
+
+### Lists with code
+
+- **Provide complete code** snippets as a separate code block
+- **Explain** what the code does **before** the **code block**
+- Put **conditional clauses before instructions**
+- Use [imperative mood](https://en.wikipedia.org/wiki/Imperative_mood) (implicit you)
+
+>- **Recommended**:
+
+1. Install `nvm`:
+
+```bash
+brew update && brew install nvm
+```
+
+_Follow the post install instructions in the success message._
+
+1. Install `node 8.10.0` (this also installs `npm`):
+
+````bash
+nvm install 8.10.0
+````
+
+>- **Not recommended**:
+
+1. You can install nvm by running `brew update` and `brew install nvm`. Once that's finished, run `nvm install 8.10.0`
+
+## Language and grammar
+
+- **Write** in **second person** (you instead of we)
+  - Make it clear to the reader who you expect them to be (e.g. devops, front end, product manager)
+  - **Not recommended**
+    - _If we're deleting multiple entries at a time ..._
+  - **Recommended**
+    - _If you're deleting multiple entries at a time ..._
+    - _When deleting multiple entries at a time ..._
+- **Explain** things **in terms of what to do** rather than what not to do (positive voice)
+  - **Not recommended**: N won't happen, if you don't XYZ.
+   - **Recommended**: To make N happen, do XYZ.
+- **Clarify antecedents**
+  - **Not recommended**: If you use the term "green beer" in an ad, then make sure it's targeted.
+  - **Recommended**: If you use the term "green beer" in an ad, then make sure **the ad** is targeted.
+- **Avoid** misplaced modifiers
+  - **Not recommended**: Developers only need to apply for one token.
+  - **Recommended**: Developers need to apply for only one token.
+- **Avoid** abbreviations
+  - **Not recommended**: js, ui, ux, VFS
+  - **Recommended**: JavaScript, user interface, user experience, veteran facing tools
+- Use [active voice](https://en.wikipedia.org/wiki/Active_voice)
+ - **Not recommended**: The extension must be registered.
+ - **Recommended**: You must register the extension.
+- Use [serial commas](https://en.wikipedia.org/wiki/Serial_comma#Creating_ambiguity)
+
+## Images
+
+- Prepend image names with document name
```

Comment:
some of these things we didn't do so we'll have a little clean up to do to make docs consistent with this style guide